projboard/tests/test_user.py

```diff
@@ -38,14 +38,24 @@ def test_get_user_by_nickname(self, generate_user):
         assert generate_user.password == user.password
 
     @pytest.mark.django_db
-    def test_get_user_by_nickname_fail(self):
-        # Tests that the user is not in the DB
-        user = User.get_user_by_nickname(NICKNAME_FAIL)
-        assert user not in User.objects.all()
+    def test_get_not_existed_user_by_nickname(self):
```

Comment:
Please take a look here,
link to learn about testing with exceptions

    with pytest.raises(User.DoesNotExist*/, match=''content oif error message"*/):
       User.get_user_by_nickname(NOT_EXISTED_USER)

Reply:
@Yarboa Done. 
In this case I used `match="User matching query does not exist."` same as the assertion error I got after running pytest.